README.md

```diff
@@ -61,8 +61,8 @@ There is only an `Octo <object> <action> [arguments]` command:
 | gist | list | [repo] [key=value]*<br>[Available keys](https://cli.github.com/manual/gh_gist_list):  `repo`\|`public`\|`secret`<br>Mappings:<br>`<CR>`: Append Gist to buffer<br>`<C-b>`: Opens Gist in web browser |
 | comment | add | |
 | | delete | |
-| | resolve | |
-| | unresolve | |
+| thread | resolve| Mark a review thread as resolved |
+| | unresolve | Mark a review thread as unresolved |
```

Comment:
Ah wait, this `README.md [RIGHT] (65,65)` is a virtual text. I can't change it. It seems to be the reference what the comment relates to. But it is hard to realize. And it is kinda annoyingly here after the text I type... Could we improve that?
Btw: is it possible to customize the sign that marks a line with a comment?

Reply:
> Ah wait, this README.md [RIGHT] (65,65) is a virtual text. I can't change it. It seems to be the reference what the comment relates to. But it is hard to realize. And it is kinda annoyingly here after the text I type... Could we improve that?

The cursor should be initially located below the virtual text line and the empty lines above should be trimmed. This may not be obvious though.

I can use an overlay instead of regular virtual text so at least this text will be fixed at a given position and not after the input text

Any ideas how to make more clear that `README.md [RIGHT] (65,65)` is a "header" maybe start with `//` and then use Comment highlght?

> Btw: is it possible to customize the sign that marks a line with a comment?

Not yet, but should be easy to support

Reply:
You can try overriding `octo_comment` sign as in:

```
sign define octo_comment text=❯ texthl=OctoNvimCommentLine linehl=OctoNvimCommentLine
```

Reply:
> You can try overriding octo_comment sign as in:

Yes, I found that. I just opened a issue yesterday at neovim/neovim#14036 to have some better sign default handling for plugins in general. In my case it should just work as I source the Octo plugin configuration after it gets loaded as `opt` package. But it could be better. :grimacing:

Reply:
> Any ideas how to make more clear that README.md [RIGHT] (65,65) is a "header" maybe start with // and then use Comment highlght?

What is your intention with this header? Maybe this would help to help. :laughing: 
I don't know how your filenames/-paths work but eventually I would put this information just in the file name?

Reply:
Conversation closable as the topic continues in the regular comments below.